doc/console/SONiC-Console-Switch-High-Level-Design.md

```diff
@@ -105,6 +131,7 @@ Console Switch enable user to manage routers/switchs via console port.
 
 1. Support configuration using SONiC CONFIG_DB
 1. Support configuration using SONiC MINIGRAPH
+1. Support configuration using SONiC config CLI
```

Comment:
Should we also support configuring the device name?

Reply:
Also we should add a statement here that the line state should be persisted in app_db

Reply:
If you means the remote devicename, the HLD mentioned it in: https://github.com/Azure/SONiC/blob/58110f88b2ffb9d2fa92934aea8fb6f20d4d2cc5/doc/console/SONiC-Console-Switch-High-Level-Design.md#33122-update-remote-device-name-for-a-console-port

For state persisting, we are using STATE_DB instead of APP_DB, I will highlight this here